BinaryObjectScanner/Protection/AlphaROM.cs

```diff
@@ -84,5 +88,50 @@ public class AlphaROM : IExecutableCheck<PortableExecutable>
 
             return null;
         }
+        
+        public string? CheckISO(string file, ISO9660 iso, bool includeDebug)
```

Comment:
For this an other implementations of the interface, please use the explicit `/// <inheritdoc/>` above each. It's not technically needed but I find it helpful to indicate methods that are from a base class or interface.

Reply:
In most places where you changed this you also got rid of the newline above. Please just do a quick sweep and add back the newline between the `inheritdoc` line and the end of the method above it.

Reply:
Damn, I was trying to keep the newline. I'll go fix that.